src/ddmd/expression.d

```diff
@@ -8520,6 +8520,7 @@ extern (C++) final class DotIdExp : UnaExp
 {
     Identifier ident;
     bool noderef;       // true if the result of the expression will never be dereferenced
+    bool wantsym;       // do not replace Symbol with its initializer during semantic()
```

Comment:
"during semantic()" - is that exact? What I mean is: the way it's written now allows replacing Symbol with its initializer in semantic2() ?